client/state/timezones/reducer.js

```diff
@@ -27,8 +32,15 @@ export const byContinents = createReducer( {}, {
 	[ TIMEZONES_RECEIVE ]: ( state, actions ) => ( actions.byContinents )
 }, continentsSchema );
 
+export const requesting = createReducer( false, {
+	[ TIMEZONES_REQUEST ]: () => true,
```

Comment:
Hmm, looks like this never reaches the reducer. Is this expected when using the data-layer @dmsnell ?